assets/js/script.js

```diff
@@ -2,11 +2,14 @@ var baseYouTubeURL = "https://www.googleapis.com/youtube/v3/search?key=AIzaSyAS3
 var baseIMDBURL = "http://www.omdbapi.com/?apikey=9e1ba2cf&";
 let movieTitleText = JSON.parse(window.localStorage.getItem("movieTitleText")) || [];
 
+//This code is called on startup, when the document is ready
 $(document).ready(function () {
+    getMovieList();
```

Comment:
@pherpat I moved the getMovieList() to the $(document).ready(function() { so that it wouldn't be all by itself, and that all of the functions that would be called when the document was ready would be together.